### ISSUE TITLE
Interactive tracing disturbs the CMS console stack

### ISSUE THREAD

**Issue by BobBolch** [adesutherland/CMS-370-BREXX#57]
In intractive tracing mode, BREXX is reading from the CMS console stack. In intractive debugging mode, it must read directly from the user trminal and leave the stack undisturbed. The CMS interface to read directly from the virtual console was not available in VM/370. I will add the required support to the RDTERM API so that BREXX can use it.

**Comment by rvjansen** [comment]
yes, this has bitten me too. Fortunately, one minute later I remembered this issue.

**Comment by RossPatterson** [comment]
This appears to have been fixed by bREXX commit ["F0047 / v0.9.12"](05540f9b8a8bedd778636ff783894f9159d0ac32), GCCLIB commit ["F0051"](5bab96cdde0b96bc5090a43ae5efbe554d8064aa) and VM/CE 1.1 CMS update HRC317DS between 2021-01-13 and 2021-03-14.
